src/adapters/rubicon.js

```diff
@@ -99,59 +99,70 @@ var RubiconAdapter = function RubiconAdapter() {
   }
 
   /**
-   * Create a (successful) bid for a unit,
+   * Create (successful) bids for a unit,
    * based on the given response
    * @param {String} placement placement code/unit path
    * @param {Object} response the response from rubicon
    * @return {Bid} a bid objectj
    */
-  function _makeBid(response, ads) {
+  function _makeBids(response, ads) {
 
     // if there are multiple ads, sort by CPM
     ads = ads.sort(_adCpmSort);
 
-    var bidResponse = bidfactory.createBid(1);
-    var ad = ads[0];
-    var size = ad.dimensions;
+    var bidResponses = [];
 
-    if (!size) {
-      // this really shouldn't happen
-      utils.logError('no dimensions given', RUBICON_BIDDER_CODE, ad);
-      return _errorBid(response, ads);
-    }
+    ads.forEach(function(ad) {
 
-    bidResponse.bidderCode = RUBICON_BIDDER_CODE;
-    bidResponse.cpm = ad.cpm;
+      var bidResponse,
+          size = ad.dimensions;
 
-    // the element id is what the iframe will use to render
-    // itself using the rubicontag.renderCreative API
-    bidResponse.ad = _creative(response.getElementId(), size);
-    bidResponse.width = size[0];
-    bidResponse.height = size[1];
+      if (!size) {
+        // this really shouldn't happen
+        utils.logError('no dimensions given', RUBICON_BIDDER_CODE, ad);
+        bidResponse = _errorBid(response, ads);
+      } else {
+        bidResponse = bidfactory.createBid(1);
 
-    // DealId
-    if (ads.deal !== undefined && ads.deal !== "") {
-      bidResponse.dealId = ads.deal;
-    }
+        bidResponse.bidderCode = RUBICON_BIDDER_CODE;
+        bidResponse.cpm = ad.cpm;
 
-    return bidResponse;
+        // the element id is what the iframe will use to render
+        // itself using the rubicontag.renderCreative API
+        bidResponse.ad = _creative(response.getElementId(), size);
+        bidResponse.width = size[0];
+        bidResponse.height = size[1];
+
+        // DealId
+        if (ads.deal !== undefined && ads.deal !== "") {
```

Comment:
I know you just copied this code, but I think it would be better to change it to `if (ads.deal) {` rather than checking for undefined and an empty string.